### DISCUSSION TITLE
00-Azure-Free-Tier

### DISCUSSION THREAD

**Discussion by giscus** [livialima/linuxupskillchallenge#9] [General]
# 00-Azure-Free-Tier

A month-long course aimed at those who aspire to get Linux-related jobs in the industry - junior Linux sysadmin, DevOps-related work, and similar. Learn the skills required to sysadmin a remote Linux server from the commandline.

https://linuxupskillchallenge.org/00-Azure-Free-Tier

**Comment by AhmadMWaddah** [comment]
So Amazing lesson with, very easy and good to understand and follow the instruction. I am Grateful for Authors.

**Comment by OTOverseNG** [comment]
This is a very amazing course

**Comment by SAILOHITC961** [comment]
As a beginer ,who only has basic knowledge of linux ,i felt these steps little hard to follow ,had to use Chatgpt along side to know what to do where.but it helps ,but can be optimized more

**Comment by systems-jackal** [comment]
Had challenges with Linux administration but i think i will have a good experience from the lessons i will get from here
